docs/docs/ref/rows_cols.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'columns/1'
+package: ALS Library
+group: Formatting
+predicates:
+- {sig: 'columns', args: {
+    1: 'Output ListOfRows as a minimal columnar array of terms',
+    2: 'Like columns/1, outputting to a given stream',
+    3: 'Output ListOfRows as a columnized rectangular array of terms'
+  }}
+---
+## FORMS
+
+`columns(ListOfRows)`
+
+`columns(ListOfRows, OutputStream)`
+
+`columns(ListOfRows, ColWidths, OutputStream)`
+
+## DESCRIPTION
+
+**`columns/1`** If ListOfRows is a list of lists of terms (Rows), and  
+    if all of the Rows are of the same length, outputs a  
+    rectangular array of the terms appearing in the  
+    Rows to user_output. The array is split into columns of minimal  
+    size to contain the appropriate terms, with one character  wide  
+    blank separators  
+
+**`columns/2`** If ListOfRows is a list of lists of terms (Rows), and  
+    if all of the Rows are of the same length, outputs a  
+    rectangular array of the terms appearing in the  
+    Rows to OutputStream. The array is split into columns of minimal  
+    size to contain the appropriate terms, with one character  wide  
+    blank separators.  
+    Note: if arg(2) is passed a list of integers(ColWidths), then  
+    columns(ListOfRows, ColsWidths user_output)  
+    is invoked.  
+
+**`columns/3`** If ListOfRows is a list of lists of terms (Rows), and  
+    if all of the Rows are of the same length, and if ColWidths  
+    is a list of positive integers of the same length as the  
+    common length as the individual lists in ListOfRows, outputs a  
+    rectangular array of the terms appearing in the Rows to  
+    OutputStream. The array is split into columns of the size of  
+    the corresponding integer of ColWidths, with one character-wide  
+    blank separators.  Terms in Rows which are of size > the  
+    corresponding integer of ColWidths are trucated.  
+
+## EXAMPLES
```

Comment:
Nice example.  Added a modified version showing what can be done with u(...)

Reply:
Would have preferred to see `city_table` (or something similar) replace `silly_table`, because `silly_table` is "random data", not "toy data", but not a big deal.